src/data/sym/basic.lean

```diff
@@ -237,13 +266,48 @@ by simp [sym.map]
   (a :: s).map f = (f a) :: s.map f :=
 by simp [map, cons]
 
+@[congr] lemma map_congr {β : Type*} {f g : α → β} {s : sym α n} (h : ∀ x ∈ s, f x = g x) :
```

Comment:
Can you pull out `β`, `s`, `decidable_eq α` and maybe `f` to the relevant `variables` declarations?

Reply:
I agree with `β`, but probably wouldn't move the others.